streams/src/main/java/org/apache/kafka/streams/TopologyConfig.java

```diff
@@ -53,36 +59,42 @@
     private static final ConfigDef CONFIG;
```

Comment:
Maybe make a note in the javadocs that settings values for anything in TopologyConfig will result in that overriding the value set in the StreamsConfig passed in to the KafkaStreams. The current javadocs kind of hint at it but they're more focused on named topologies so I don't think it's really that clear

Reply:
Actually, I think you already made it very clear in the javadoc [here](https://github.com/apache/kafka/blob/trunk/streams/src/main/java/org/apache/kafka/streams/processor/internals/namedtopology/TopologyConfig.java#L46-L51):
 
> Streams configs that apply at the topology level. The values in the {@link StreamsConfig} parameter of the {@link org.apache.kafka.streams.KafkaStreams} or {@link KafkaStreamsNamedTopologyWrapper} constructors will determine the defaults, which can then be overridden for specific topologies by passing them in when creating the topology builders via the {@link org.apache.kafka.streams.StreamsBuilder()} method.

Please let me know if you think we still need to add anything. Thanks.